.github/workflows/blackduck.yaml

```
@@ -3,6 +3,9 @@ name: Black Duck scan
 on:
   schedule:
     - cron: 45 4 15 * *
+  push:
+    branches:
+      - detect_accuracy
```

This wont work on main, only if we push to detect_accuracy
```suggestion
      - main
```

---

Good catch! I hadn't pushed but now it should be all good